packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -159,6 +160,7 @@ describeNoCompat("Container", (getTestObjectProvider) => {
     [
         { eventName: "fluid:telemetry:DeltaManager:GetDeltas_Exception", error: "expectedFailure" },
         { eventName: "fluid:telemetry:Container:ContainerClose", error: "expectedFailure" },
+        { eventName: "fluid:telemetry:Container:ContainerClose", error: "expectedFailure", disposed: true },
```

Comment:
Does this mean we'll expect the container to raise two distinct ContainerClose events? Would it be better for the new one to be ContainerDisposed? I think our telemetry will get confusing otherwise.

Reply:
It's just so we align the naming. I agree that having a distinct event would be clean, but most people probably shouldn't have to know it can exist in place of ContainerClose.
The idea is that there can be situations where consumers of Container can just call "dispose", which technically also closes the container. So, we should log an event that is recognizable as a ContainerClose event without knowing these implementation details (could be done as "ContainerClose_Disposed" or with telemetry prop).

Reply:
My concern is if calling dispose() generates _two_ ContainerClose events, one with no extra `disposed` prop and one with it. That's what I'd find confusing. Or if someone calls close first and later calls dispose once they've extracted everything they need from the container and are ready to fully get rid of it, that'd also result in two ContainerClose events, right?

Reply:
@alexvy86 Thanks for triggering this discussion. I brought it up with others and the consensus is to go with "ContainerDispose"

Reply:
One further piece of feedback on that front: several of the tests that now expect the extra event are named in terms of "closing the container", e.g. "A large op will close the container". With the new distinction we're making, I think it's a good idea to update their names to be explicit about expectations of a container just closing VS a container closing and being disposed.